pkg/scheduler/algorithmprovider/defaults/defaults_test.go

```diff
@@ -43,11 +44,13 @@ func TestCopyAndReplace(t *testing.T) {
 			expected:    sets.String{"A": sets.Empty{}, "B": sets.Empty{}},
 		},
 	}
-	for _, testCase := range testCases {
-		result := copyAndReplace(testCase.set, testCase.replaceWhat, testCase.replaceWith)
-		if !result.Equal(testCase.expected) {
-			t.Errorf("expected %v got %v", testCase.expected, result)
-		}
+	for i, testCase := range testCases {
+		t.Run(fmt.Sprintf("%v/replace %v with %v", i, testCase.replaceWhat, testCase.replaceWith), func(t *testing.T) {
```

Comment:
I think these tests could have explicit names like 'replace existing item' and 'replace nonexistent item'